### ISSUE TITLE
Set default demo account to noah@demo.dev

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/twenty#7304]
# Current behavior

Today, the default account on demo.twenty.com is set to tim@apple.dev, like we do in local

https://github.com/user-attachments/assets/2f5d7f44-79af-494a-a265-61ac7c04cd69

# Desired Behavior

We want the default demo default login to be set to:

email: noah@demo.dev
password: Applecar2025

**Comment by charlesBochet** [comment]
@FelixMalfait @vijaysingh2219 I think there was a misunderstanding on this one: 
- we want the credentials in the README to be noah@demo.dev
- AND the modal to be prefilled with noah@demo.dev too

**Comment by FelixMalfait** [comment]
@charlesBochet it's simpler to have a single credential for everyone. It works perfectly today now that the readme has been updated, demo env is autofilled and everyone's happy :)

**Comment by vijaysingh2219** [comment]
> @FelixMalfait @vijaysingh2219 I think there was a misunderstanding on this one:
> 
> * we want the credentials in the README to be [noah@demo.dev](mailto:noah@demo.dev)
> * AND the modal to be prefilled with [noah@demo.dev](mailto:noah@demo.dev) too

Hey @FelixMalfait , I believe my changes were correct, as the email should be noah@demo.dev both in the README and in the prefilled modal. However, the README was changed to tim@apple.dev instead.

**Comment by FelixMalfait** [comment]
@vijaysingh2219 your change was indeed correct for the issue yes thanks, but I changed the goal. We agreed with @Bonapara it's simpler to have a single set of credentials

**Comment by vijaysingh2219** [comment]
> @vijaysingh2219 your change was indeed correct for the issue yes thanks, but I changed the goal. We agreed with @Bonapara it's simpler to have a single set of credentials

Thanks for the update, @FelixMalfait! I appreciate the clarification about the goal.

**Comment by Bonapara** [comment]
Thanks @vijaysingh2219!

**Comment by priyankchopra2** [comment]
Hi @Bonapara and team, today I was trying to set up the project locally and was following the docs from https://twenty.com/developers/local-setup. 

Everything was working fine until I tried to login as mentioned in the docs with tim@apple.dev email but I was not able to login (attaching screenshot of the error for your reference). 

<img width="500" alt="Error snip" src="https://github.com/user-attachments/assets/68cbbc43-80d3-47d2-9c96-6b47bab7bb4a">


Then I checked this issue and tried to logging in with noah@demo.dev and it worked!


<img width="500" alt="Logged In snip" src="https://github.com/user-attachments/assets/1c2b72e4-5d09-4fe2-a9f6-b68a7196b585">


To ensure a seamless experience for new contributors, I just wanted to highlight the login issue that needs to be corrected, either in the documentation or the application.
